cypress/component/BackendHealthCheck.cy.tsx

```diff
@@ -0,0 +1,6 @@
+describe('BackendHealthCheck.cy.tsx', () => {
```

Comment:
Do we need test that does not test anything?

Reply:
Just a placeholder / sample